spring-rabbit/src/main/java/org/springframework/amqp/rabbit/config/NamespaceUtils.java

```diff
@@ -48,10 +48,10 @@
  */
 public abstract class NamespaceUtils {
 
-	static final String BASE_PACKAGE = "org.springframework.amqp.core.rabbit.config";
-	static final String REF_ATTRIBUTE = "ref";
-	static final String METHOD_ATTRIBUTE = "method";
-	static final String ORDER = "order";
+	protected static final String BASE_PACKAGE = "org.springframework.amqp.core.rabbit.config";
+	protected static final String REF_ATTRIBUTE = "ref";
+	protected static final String METHOD_ATTRIBUTE = "method";
+	protected static final String ORDER = "order";
```

Comment:
I'd make these `public` as well. Some Framework extensions may be interested in the reuse of them.